app/views/crowdblog/admin/posts/index.html.slim

```diff
@@ -8,12 +8,14 @@
       ul.nav.nav-pills
         li class="#{@state ? '' : 'active'}"
           = link_to 'All', crowdblog.admin_posts_path
+        li class="#{@state == 'drafted' ? 'active' : ''}"
```

Comment:
you could move this comparison to helper ... or javascript function or somewhere else, just to clean the view ^^